src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -234,6 +247,13 @@ interface IRewardsCoordinator {
         uint256 claimedAmount
     );
 
+    event PerformanceBasedRewardCreated(
+        address indexed avs,
+        uint256 indexed submissionNonce,
+        bytes32 indexed rewardsSubmissionHash,
+        PerformanceRewardsSubmission rewardsSubmission
```

Comment:
Can we emit the reward type here too?

Reply:
Yessir.

Reply:
Should we also be doing this for the other events, and while we're on the topic I feel like event names for reward submissions should be more consistent since they all serve the same purpose, generally.

Reply:
Let's chat about this one tmrw?